src/panels/config/devices/device-detail/integration-elements/zwave_js/ha-device-actions-zwave_js.ts

```diff
@@ -29,34 +28,36 @@ export class HaDeviceActionsZWaveJS extends LitElement {
 
   @state() private _entryId?: string;
 
-  @state() private _nodeId?: number;
-
   @state() private _node?: ZWaveJSNodeStatus;
 
-  protected updated(changedProperties: PropertyValues) {
+  public willUpdate(changedProperties: PropertyValues) {
     if (changedProperties.has("device")) {
```

Comment:
Always call super when overriding.
```suggestion
    super.willUpdate(changedProperties);
    if (changedProperties.has("device")) {
```